src/main/java/main/model/db/dao/project/UserDao.java

```diff
@@ -39,6 +67,36 @@ public UserDto IsAuthorized(String sessionId) throws AqualityException {
         else{
             throw new AqualityPermissionsException("Credentials you've provided are not valid. Reenter please.", user);
         }
+
+        ProjectUserDto projectUserDto = new ProjectUserDto();
+        projectUserDto.setUser_id(user.getId());
+        user.setProjectUsers(new ProjectUserController(user).getProjectUserForPermissions(projectUserDto));
         return user;
     }
+
+    /**
+     * Check If Token Valid
+     * @param sessionId Session ID = project:{projectId}:{token}
+     * @return Project Engineer
```

Comment:
does it only Project Engineer or it could be f.e. just Viewer?

Reply:
User created via API Token is only Project Engineer